README.md

```diff
@@ -8,4 +8,23 @@ To build this, you need the following:
 - [`latexmk`](https://ctan.org/pkg/latexmk?lang=en) which may come with your LaTeX distribution
 - [Graphviz](https://graphviz.org/)
 
-After resolving these dependencies, run `make` to compile `spec.md` into `build/spec.pdf`.
+For compatability with continuous integration (CI) testing, use the versions of
+`pandoc` and `pandoc-crossref` that are [listed in the CI GitHub
+Action](.github/workflows/continuous-integration-ci.yml). If this release is
+*not* available in your package manager, you can download binaries from their
+GitHub releases pages:
+- [`pandoc` releases](https://github.com/jgm/pandoc/releases)
+- [`pandoc-crossref` releases](https://github.com/lierdakil/pandoc-crossref/releases)
+
+To run tests, you need Verilator and `firtool` available on your `$PATH`.
```

Comment:
```suggestion
To run tests, you need Verilator and `firtool` available on your `PATH`.
```
Maybe?

Reply:
Fixed in: https://github.com/chipsalliance/firrtl-spec/pull/320/commits/a1821d0f76c083f306a1993562a57f16cf1a8d64